mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMessageSearchIndexTest.java

```diff
@@ -150,18 +150,4 @@ public void multimailboxSearchShouldReturnUidOfMessageMarkedAsSeenInTwoMailboxes
     public void multimailboxSearchShouldReturnUidOfMessageMarkedAsSeenInAllMailboxes() throws MailboxException {
     }
 
-    @Ignore
```

Comment:
These lines prevented the test code from the super interface to run. Now they do run.

By removing these lines, I activate the test ;-)